packages/core-js/test/utils/comparator.spec.js

```diff
@@ -0,0 +1,189 @@
+const { equal } = require('assert/strict');
+const { compareStorageStructs } = require('../../utils/comparator');
+
+describe('utils/comparator.js', function () {
+  const previousStructsMap = [
+    {
+      contract: {
+        name: 'GlobalNamespace',
+        id: 614,
+      },
+      struct: {
+        name: 'GlobalStorage',
+        members: [
+          {
+            name: 'value',
+            type: 'uint256',
+          },
+          {
+            name: 'someValue',
+            type: 'uint256',
+          },
+        ],
+      },
+    },
+    {
+      contract: {
+        name: 'OwnerNamespace',
+        id: 630,
+      },
+      struct: {
+        name: 'OwnerStorage',
+        members: [
+          {
+            name: 'owner',
+            type: 'address',
+          },
+          {
+            name: 'nominatedOwner',
+            type: 'address',
+          },
+        ],
+      },
+    },
+    {
+      contract: {
+        name: 'ProxyNamespace',
+        id: 644,
+      },
+      struct: {
+        name: 'ProxyStorage',
+        members: [
+          {
+            name: 'implementation',
+            type: 'address',
+          },
+        ],
+      },
+    },
+    {
+      contract: {
+        name: 'SettingsNamespace',
+        id: 658,
+      },
+      struct: {
+        name: 'SettingsStorage',
+        members: [
+          {
+            name: 'aSettingValue',
+            type: 'uint256',
+          },
+        ],
+      },
+    },
+  ];
+
+  describe('compareStorageStructs no updates', function () {
+    it('detects no changes when both structMaps are equal', function () {
+      let currentStructsMap = JSON.parse(JSON.stringify(previousStructsMap));
```

Comment:
@leomassazza, What do you think of doinga `clone()` util for `JSON.parse(JSON.stringify(previousStructsMap))`?

Reply:
done